main.py

```diff
@@ -1,10 +1,53 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json") as f:
+        players = json.load(f)
+
+    for player in players:
```

Comment:
Omit calling for `players[player]` several times in a following way
```suggestion
    for nickname, player_data in players.items():
```